client/webpack.config.js

```diff
@@ -46,41 +43,46 @@ module.exports = env => {
                   modules: true,
                   localIdentName: "[local]--[hash:base64:5]",
                   minimize: PROD,
-                  sourceMap: !PROD,
-                },
+                  sourceMap: !PROD
+                }
               },
               {
                 loader: "postcss-loader",
                 options: {
                   ident: "postcss",
-                  plugins: (loader) => [
+                  plugins: loader => [
                     require("postcss-import")(),
                     require("postcss-url")(),
```

Comment:
Note: eslint is complaining about `postcss-url`

Reply:
weird that with my previous PR, eslint didn't complain ...